### PR TITLE
Add quietZone prop

### PR DESCRIPTION
**Pull request by sseppola**
This adds support for a `quietZone` prop.

The diff may be a bit difficult to read because I ran `npm standard --fix` to make the pre-commit hook pass, however it made changes to every line, so I recomend you just view [this commit](https://github.com/awesomejerry/react-native-qrcode-svg/commit/5c87c67876abd03406c0749f4ea758adf32be849).

The main changes are:
- Wrap everything in a parent `<Svg>`
- Fix the size of the qr to 100
- Scale the QR using the `viewBox` attribute

This is the best guide to `viewBox` I know of: https://www.sarasoueidan.com/blog/svg-coordinate-systems/#viewbox-syntax

In short:
```jsx
<Svg viewBox={[
  -quietZone,  // move origin-x coordinate "outside" the default viewbox
  -quietZone,  // move origin-y coordinate "outside" the default viewbox
  QR_SIZE + quietZone * 2,  // increase width
  QR_SIZE + quietZone * 2   // increase height
].join(' ')}
/>
```
This leaves half the quiet zone around the whole QR code.

### CODE REVIEW

**Comment by giautm** [comment]
@sseppola : Hi, This feature cause a bug on Android, that QrCode not rendered anymore. Please check anh revert it.

```
"expo": "^35.0.0",
"react": "16.8.3",
"react-native-qrcode-svg": "^5.3.2",
"react-native-svg": "~9.9.2",
```
